blocks.md

```diff
@@ -338,6 +338,40 @@ Key | Values | Required | Default
 `graph_down` | Display a bar graph for download speed. | No | `false`
 `interval` | Update interval, in seconds. | No | `1`
 
+## Nvidia Gpu
+
+Proprietary nvidia driver required.
+
+Creates a block which displays the Nvidia GPU utilization, temperature, used and total memory, fan speed, gpu clocks. You can set gpu label, that displayed by default.
+
+Clicking the left button on the icon changes the output of the label to the output of the gpu name. Same with memory: used/total. 
+
+Clicking the left button on the fans turns on the mode of changing the speed of the fans using the wheel. Press again to turn off the mode. For this opportunity you need nvidia-settings!
+
+### Examples
+
+```toml
+[[block]]
+block = "nvidia_gpu"
+label = "GT 1030"
+memory = false
+clocks = true
+interval = 1
+```
+
+### Options
+
+Key | Values | Required | Default
+----|--------|----------|--------
+`gpu_id` | GPU id in system | No | 0
+`label` | Display custom gpu label | No | ""
+`interval` | Update interval, in seconds. | No | `1`
+`utilization` | Display gpu utilization. In percents. | No | `true`
```

Comment:
Should these be `show_utilization`, `show_memory`, `show_temperature`, `show_fan_speed`, and `show_clocks` instead? That would fit the style of other blocks more closely.

Reply:
I have not seen this in other blocks, but this is logical.

Reply:
I should clarify: the style we *aspire* to with other blocks. Thanks for the changes!